src/phoenix/datasets/dataset.py

```diff
@@ -328,6 +328,15 @@ def _parse_dataframe_and_schema(dataframe: DataFrame, schema: Schema) -> Tuple[D
             unseen_column_names,
         )
 
+    for schema_field_name in ["prompt_column_names", "response_column_names"]:
```

Comment:
```suggestion
    for schema_field_name in LLM_SCHEMA_FIELD_NAMES:
```
Can be in a constants list

Reply:
Resolved in [87c4ed8](https://github.com/Arize-ai/phoenix/pull/609/commits/87c4ed8b0c2db8dcf09f42e72d566d3bbc56dc93)